README.adoc

```diff
@@ -7,6 +7,8 @@ It allows compilation of languages such C, {cpp}, Objective-C and Objective-{cpp
 
 *For more information, visit the link:https://nokee.dev[official project homepage]*
 
+image:https://img.shields.io/badge/Revved%20up%20by-Gradle%20Enterprise-06A0CE?logo=Gradle&labelColor=02303A[link=https://ge.nokee.dev/scans]
```

Comment:
❓ What is the meaning behind "Revved up" in the badge? Would there be a smaller alternative to the badge, maybe text only or something else? I'm not personally a big fan of badges; as you can see, there is none on the project, and that is by design. I wouldn't mind adding a link to the build scan, and I'm not sure if the badge is the best solution.

Reply:
This is the standard badge that we use for all open source projects which we sponsor with Gradle Enterprise.
For example, [Gradle](https://github.com/gradle/gradle/blob/master/README.md), [Spring](https://github.com/spring-projects/spring-boot/blob/main/README.adoc#spring-boot---) and [Jetbrains](https://github.com/JetBrains/kotlin/blob/master/ReadMe.md) all have the badge in the readme in addition to all the projects [listed on our OSS projects page](https://gradle.com/enterprise-customers/oss-projects/).

It helps contributors know where to look to find the Build Scans associated with the Nokee project and also shows our support that we are providing to the Nokee project.